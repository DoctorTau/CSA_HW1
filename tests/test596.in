454
3844 3599 -6185 1677 3026 8338 3884 989 9576 2129 9938 5704 -9598 8476 2293 8542 -2718 -8088 3091 -5265 -753 -7450 -3477 8450 -1724 -6010 1514 -9157 -9454 6908 -5176 5476 -434 9875 6560 -6349 -6042 928 -7472 -1744 7437 1858 -9939 -1655 -8990 -1247 -8561 9320 1510 8588 -135 4337 5120 -4776 -7232 -5427 -2178 3491 -195 -7718 6479 -7098 -1855 8601 -5541 7387 9638 -9125 -266 6135 4187 -6477 3067 -2071 1841 -9604 -9380 -2140 -2897 1453 6123 1021 -6367 1373 3108 3416 9845 7945 -3585 -5096 -4285 -5166 -7927 5426 -332 -6287 578 8843 -9772 -7994 -2249 8333 2129 2887 -558 -3699 -1927 -6785 8651 -5007 4840 2024 -9774 -9128 -8322 678 -1775 5430 7341 -6245 -6459 -8748 -7538 1002 -3826 -3687 7137 -1033 -3391 272 -742 -161 7351 1951 -4461 -3303 8746 -1832 -9542 -8089 -3898 -7255 6181 -6356 -4875 -8500 -2437 -3097 5055 9783 -2170 -6581 -485 4999 -5518 -5499 8466 -5441 6797 6441 -921 5714 -4980 -2084 5279 7683 -5838 2236 -5438 7898 2501 705 -8198 -9576 6356 -1576 -8445 -9773 -4675 -47 6520 4291 -5294 9801 -5635 5467 -5232 -9461 5157 -9803 6305 -5956 -2321 -9183 3975 8401 -7146 1670 -9754 -3798 5777 7572 -7132 -1461 8459 -5249 5539 -6300 8013 -7856 4539 -3056 5804 -6752 -3601 -9748 -8612 -8270 -1998 4597 8803 952 -7812 -5419 -1086 2949 -2505 -5236 1191 7788 6913 4080 -636 4759 -5583 6449 5402 -1081 -5957 205 -5630 759 -8263 -7945 -2549 9992 -6667 -9190 4631 5805 562 -6130 -4370 9310 9697 -4827 -3839 5716 692 8002 3989 -6979 -991 -7237 -4539 638 6939 7233 -5175 -8207 3804 -5600 7284 726 6397 -9975 367 -4613 -3111 -3005 770 6640 -337 -4600 9701 7385 8486 -5635 -4184 2204 7897 9370 2269 6574 -9491 1181 36 5505 8465 3409 9952 -5316 1397 4466 -9850 -7309 7031 -5816 7990 -7032 8909 3873 -1849 -3397 8323 7530 4668 6729 -53 -7199 8171 -7661 -8480 -6376 3851 3453 -6751 -7891 -1712 -9961 1879 710 -5153 815 6353 5108 2924 8773 -3997 1122 7193 -8429 -4467 -9877 -9494 -392 -2709 -6365 -7020 6507 -9921 9264 2715 9359 -1278 2413 9713 -6503 6208 -1295 1022 -9489 -398 -5282 -2396 -884 -1909 -201 -8711 4242 -3747 -7635 -7053 -5729 -4403 2283 -3300 -3575 7805 48 -879 1764 6048 -4863 -5459 -6523 9142 -930 -206 -2438 793 7125 -1632 -1300 -9934 5246 -6348 9717 -7554 6491 -4966 4397 2390 -5915 4866 -2274 5757 3266 -8193 4384 -4288 -8823 4905 842 3743 1463 -2771 4397 -5473 313 8746 -4864 1046 7813 -6982 5374 7617 -3839 -4275 1850 -3270 3256 -7168 -5181 -2913 5115 6861 5640 -7048 451 -9571 -2473 -1473 -1591 7006 -8728 6147 -6970 -6182 3903 -1636 6502 -7700 -2672 