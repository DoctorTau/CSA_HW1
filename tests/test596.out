-9598 -2718 -8088 -7450 -1724 -6010 -9454 -5176 -434 -6042 -7472 -1744 -8990 -4776 -7232 -2178 -7718 -7098 -266 -9604 -9380 -2140 -5096 -5166 -332 -9772 -7994 -558 -9774 -9128 -8322 -8748 -7538 -3826 -742 -1832 -9542 -3898 -6356 -8500 -2170 -5518 -4980 -2084 -5838 -5438 -8198 -9576 -1576 -5294 -5232 -5956 -7146 -9754 -3798 -7132 -6300 -7856 -3056 -6752 -9748 -8612 -8270 -1998 -7812 -1086 -5236 -636 -5630 -9190 -6130 -4370 -5600 -4600 -4184 -5316 -9850 -5816 -7032 -8480 -6376 -1712 -9494 -392 -7020 -1278 -398 -5282 -2396 -884 -3300 -930 -206 -2438 -1632 -1300 -9934 -6348 -7554 -4966 -2274 -4288 -4864 -6982 -3270 -7168 -7048 -8728 -6970 -6182 -1636 -7700 -2672 